core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -491,31 +491,35 @@ public synchronized boolean getUriValidationEnabled() {
   }
 
   /**
-   * Acquire a client resource from {@link #mBlockMasterClientPool} or
+   * Acquires a closeable client resource from the provided dynamic resource pool.
+   * <p>
+   * Acquires a client resource from {@link #mBlockMasterClientPool} or
    * {@link #mFileSystemMasterClientPool}.
-   *
+   * <p>
    * Because it's possible for a context re-initialization to occur while the resource is
    * acquired this method uses an inline class which will save the reference to the pool used to
    * acquire the resource.
```

Comment:
I would rewrite this paragraph like this:
"Uses an inline class which will save the reference to the pool used to acquire the resource because it's possible for a context re-initialization to occur while the resource is acquired by this method ."